io.openems.edge.application/EdgeApp.bndrun

```diff
@@ -31,8 +28,8 @@
 	bnd.identity;id='io.openems.edge.controller.asymmetric.fixactivepower',\
 	bnd.identity;id='io.openems.edge.controller.asymmetric.fixreactivepower',\
 	bnd.identity;id='io.openems.edge.controller.asymmetric.phaserectification',\
+	bnd.identity;id='io.openems.edge.controller.byd.alarm',\
```

Comment:
This channel should be renamed, because it is generic. 

I suggest **io.openems.edge.controller.io.alarm**